pyproject.toml

```diff
@@ -50,7 +50,7 @@ dependencies = [
     "urllib3==1.26.18",
     "docker==6.0.0",
     # avoid specific requests version to fix bug in docker-py
-    "requests > 2.26.0, != 2.32.0",
+    "requests==2.31.0",
```

Comment:
There is a 2.31.1. Can we set it to `<2.32.0`?

Reply:
Done